tcms/testplans/templates/testplans/mutable.html

```diff
@@ -4,16 +4,19 @@
 
 {% block head %}
     {{ form.media }}
+    {% block title %}
```

Comment:
Blocks are different from HTML tags. `title` and `head` are two separate blocks. See `base.html`. 

Putting them one inside the other doesn't accomplish anything. In fact it kind of breaks the page because the text will appear above the navigation menu. <--- verify this manually !!!

`title` block should be outside `head` block.